trajan/accessor.py

```diff
@@ -47,6 +48,40 @@ def __new__(cls, ds):
 
         tx = detect_tx_dim(ds)
 
+        # if we have a 1D dims, this is most likely some contiguous data
+        # there may be a few exceptions though, so be ready to default to the classical 2D parser below
+        if len(tx.dims) == 1:
+            # only support ContiguousRagged for now
+            ocls = ContiguousRagged
+            
+            # we have a dataset where data are stored in 1D array
+            if "index" in tx.dims:
+                obsdim = "index"
+
+                if "time" in ds.coords:
+                    timecoord = "time"
+                else:
+                    raise ValueError("cannot find timecoord in 1D-array dataset")
+
+                if "trajectory" in ds.coords:
```

Comment:
and likewise this should be the variable whose attribute `cf_role = "trajectory_id"`
I.e. it does not have to be named `trajectory`although it normally would be.

Reply:
Yes, looks good!